src/credentials/CHIPCertFromX509.cpp

```diff
@@ -765,5 +765,67 @@ CHIP_ERROR ConvertX509CertsToChipCertArray(const ByteSpan & x509NOC, const ByteS
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ConvertChipCertArrayToChipCerts(const ByteSpan & operationalCert, ByteSpan & noc, ByteSpan & icac)
```

Comment:
This does not convert, it extracts. Suggest renaming `ExtractCertsFromCertArray`

Reply:
Will update.